etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -55,14 +55,26 @@ spec:
           value: "300"
         - name: PGBOUNCER_IGNORE_STARTUP_PARAMETERS
           value: "extra_float_digits"
+        - name: SERVER_TLS_SSLMODE
```

Comment:
Should this be `PGBOUNCER_SERVER_TLS_SSLMODE`? It's currently what I have in my pgbouncer deployment

Reply:
Good catch. It works with the newer pg image we were using but I fixed it in both places.